modules/plugin/jdbc/jdbc-postgis/src/main/java/org/geotools/data/postgis/PostgisNGDataStoreFactory.java

```diff
@@ -32,11 +35,17 @@
 import org.geotools.jdbc.JDBCDataStoreFactory;
 import org.geotools.jdbc.SQLDialect;
 import org.geotools.util.KVP;
+import org.postgresql.jdbc.SslMode;
 
-/** @source $URL$ */
+
```

Comment:
Avoid any manual reformat, let the automatic formatting do the work. PR with manual reformats will be rejected.